fossunited/doctype_ids.py

```diff
@@ -12,3 +12,23 @@
 HACKATHON_PROJECT = "FOSS Hackathon Project"
 HACKATHON_TEAM = "FOSS Hackathon Team"
 HACKATHON_TEAM_MEMBER = "FOSS Hackathon Team Member"
+
+## FOSSUnited Module
+EVENT_CFP = "FOSS Event CFP"
+CFP_REVIEW = "FOSS Event CFP Review"
+CFP_REVIEWER = "FOSS Event CFP Reviewer"
+EVENT_GRANT = "FOSS Event Grant"
+FOSSUNITED_SETTINGS = "FOSSU Settings"
+STATE = "State"
+CUSTOM_ANSWER = "FOSS Custom Answer"
+CUSTOM_QUESTION = "FOSS Custom Question"
+EVENT_FIELD = "FOSS Event Field"
+EVENT_SCHEDULE = "FOSS Event Schedule"
+EVENT_SPONSOR = "FOSS Event Sponsor"
+GLOBAL_CFP_REVIEWER = "FOSS Global CFP Review Member"
+GLOBAL_REVIEW_SETTINGS = "FOSS Global CFP Review Settings"
+PROJECT_GRANT = "FOSS Project Grant"
+TEAM = "FOSS United Team"
+FOSSUNITED_SETTINGS = "FOSSU Settings"
+INDUSTRY_PARTNERS = "Industry Partners"
+ORGANIZATION = "Organization"
```

Comment:
it doesn't look like they are all being used but that's okay :shrug:

Reply:
Yes, they are not used. But I added them in case if we have to use these constants in the near future.

Reply:
`TEAM`, `INDUSTRY_PARTNERS` are used in web templates - I assumed we are not replacing constants in web templates in this PR.

`FOSSU_SETTINGS` is actually used in the workflow - which is a bit outside the scope of fossunited app in general. 
<img width="1244" alt="Screenshot 2024-08-14 at 12 21 30 PM" src="https://github.com/user-attachments/assets/4085d6ed-ac34-484e-87b2-5ad8fe8e86bc">


`ORGANIZATION` is something that I think is not in use now -I couldn't find any relevant ones on a first search - we could check again and see.

Reply:
We are only replacing in the py. please correct me if it is getting used anywhere. Locally, I had tried finding their use in any of the py files multiple times through `grep`, `xargs` and also at the end with vscode.

Reply:
Yep - web templates are html pages, hence not relevant to this PR, if we are not focusing on files other than `.py`

However, FOSSU Settings seem in `utils.py`